### PR TITLE
Add ipq40xx arm target

### PR DESCRIPTION
**Pull request by jkilpatr**
Turns out the glb1300 is a little different than the Omnia, this accounts
for that.

### CODE REVIEW

File: scripts/openwrt_build_ipq40xx.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```
Comment (drozdziak1) [inline scripts/openwrt_build_ipq40xx.sh]: Could you in fact list that this file helps with the gl-b1300?

File: scripts/openwrt_build_mvebu.sh [hunk 2]
```diff
@@ -9,12 +9,12 @@ export TOOLCHAIN=toolchain-arm_cortex-a9+vfpv3_gcc-7.3.0_musl_eabi
 export TARGET_CC=$DIR/staging_dir/$TOOLCHAIN/bin/arm-openwrt-linux-gcc
 export TARGET_LD=$DIR/staging_dir/$TOOLCHAIN/bin/arm-openwrt-linux-ld
 export TARGET_AR=$DIR/staging_dir/$TOOLCHAIN/bin/arm-openwrt-linux-ar
-export CARGO_TARGET_ARM_UNKNOWN_LINUX_MUSLEABIHF_LINKER=$TARGET_CC
-export CARGO_TARGET_ARM_UNKNOWN_LINUX_MUSLEABIHF_AR=$TARGET_AR
+export CARGO_TARGET_ARMV7_UNKNOWN_LINUX_MUSLEABIHF_LINKER=$TARGET_CC
```
Comment (drozdziak1) [inline scripts/openwrt_build_mvebu.sh]: Is this a fix for the Illegal Instruction problem you mentioned on Riot?

File: scripts/openwrt_build_ipq40xx.sh (hunk 1 above)
Comment (drozdziak1) [reply scripts/openwrt_build_ipq40xx.sh]: Is there docs for these scripts?

**Comment by mpapierski** [comment]
Looks good. I retried one build on integration tests that seemed to be stalled. Also, one build fails because it seems that we have misformatted code with `cargo +nightly fmt`. This could be addressed in a separate PR, though.

File: scripts/openwrt_build_ipq40xx.sh (hunk 1 above)
Comment (jkilpatr) [reply scripts/openwrt_build_ipq40xx.sh]: not as much as I would like.

File: scripts/openwrt_build_mvebu.sh (hunk 2 above)
Comment (jkilpatr) [reply scripts/openwrt_build_mvebu.sh]: well the new ipq40xx stuff is the fix for that, armv7 is just me realizing that rust would generate compatible but not as capable code

File: scripts/openwrt_build_ipq40xx.sh (hunk 1 above)
Comment (drozdziak1) [reply scripts/openwrt_build_ipq40xx.sh]: Looks great, thanks!

**Pull request merged**